### ISSUE TITLE
Name for monuments

### ISSUE THREAD

**Issue by Martinligabue** [streetcomplete/StreetComplete#6042]
tl:dr, exactly as board_name quest, I couldn't manage to send a push request, but I'd have tried to clone the board_name to memorial_name
### General
Affected tag(s) to be modified/added: [Key:monument](https://wiki.openstreetmap.org/wiki/Tag:historic%3Dmonument), [Key:memorial](https://wiki.openstreetmap.org/wiki/Key:memorial) and similar
Question asked: **Is there an example tag on item XY?**
https://www.openstreetmap.org/node/11007296453

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
- [x] 🚧 To be added tag is established and has a useful purpose
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [x] 🐿️ Easily answerable by any pedestrian from the outside but a survey is necessary
- [x] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
- [x] 🕓 Applies to a reasonable number of map data (Worth the effort)

### Ideas for implementation
like `/src/main/java/de/westnordost/streetcomplete/quests/board_name/AddBoardName.kt`

eventually extended to similar pois

**Comment by matkoniecz** [comment]
note: `- [x]` shows as nicely formatted checkbox, while `- [*]` does not

**Comment by westnordost** [comment]
Can you do some research on this? E.g.  - When should the `name` *not* be asked (because another tag is set which makes it unnecessary and/or wrong to tag a name)? How many elements are we talking about here? Can all monuments and memorials (types) expected to have a (signed) name?

**Comment by Vuizur** [comment]
I think it would be important to tag it with name:signed instead of noname (compared to the information board quest). And maybe to handle multilingual names like name:en.

I don't see memorials that can't have a name, at least with a quick look through the wiki.

**Comment by westnordost** [comment]
What about Stolpersteine? They are memorials FOR people, but that's not their `name`, right?

**Comment by Vuizur** [comment]
The wiki says that they should generally be tagged with name=<name of person>, for ghost bikes it is the same. But it also says that some users use memorial:name, so objects with that tag should be on the "ignore list".

**Comment by westnordost** [comment]
I am still unsure whether the monuments and memorials (in particular) that currently don't have a name don't in their majority have no (signed) name.... but on the other hand, it's of course difficult to ascertain that.

**Comment by Martinligabue** [comment]
I'm not able to use overpass or other tools like that yet, for now I rely on the wiki, I looked through the most used tags, and, at least imo, all those that I know of have a name, for some it's not written explicitly in the wild, but they still have a name.

**Comment by matkoniecz** [comment]
https://overpass-turbo.eu/s/1VGV finds monuments within name

you can navigate to are you know well and judge how many are actually named

**Comment by Vuizur** [comment]
I looked a bit into overpass (but haven't done extensive research). I think unnamed monuments are mostly mistagged memorials.

I looked into my area, there are some memorials where the name exists, but hasn't been added, but probably less than 50 percent of the results. Maybe one could ask "Does this memorial have a name?", but I'm not sure.

**Comment by Martinligabue** [comment]
In every place I have knowledge of, all the monuments are actually memorials or completely wrong, maybe it could be considered to have them checked, as I didn't find a single place where they should be kept as historic=monument.
I guess this issue should only focus on the memorials, then.

**Comment by mnalis** [comment]
>  I guess this issue should only focus on the memorials, then.

In Croatia, many types of memorials (in particular e.g. `memorial=plaque`, `memorial=cross`, `memorial=obelisk`, `memorial=stele`, and also sometimes  `memorial=war_memorial`) don't have a `name=*`. 

More often they might have have a [subject=*](https://wiki.openstreetmap.org/wiki/Key:subject) (or [subject:wikidata=*](https://wiki.openstreetmap.org/wiki/Key:subject:wikidata) etc.) sometimes, and **very often** they have [inscription=*](https://wiki.openstreetmap.org/wiki/Key:inscription)[^1]

[^1]: but much less often do people actually _record_ that `inscription` in OSM, because it is a lot to type and often can be over the 255-character limit for tag values

**Comment by HugoTHOLLON** [comment]
Hello.  
Is someone taking care of this issue ?  
Is the issue still relevant and can participate to it ?

**Comment by mnalis** [comment]
> I looked a bit into overpass (but haven't done extensive research). I think unnamed monuments are mostly mistagged memorials.

There could be `Uh....` answer like `It is is not a monument, but just a memorial` which could allow for retagging them properly. (as it is IMHO likely that users will visit `Uh...` if alleged "monument" has no name).

>  Is the issue still relevant and can participate to it ?

I'd say yes... It has label `new quest` which means it has been approved and may be implemented.  (But it is not up to me, so best wait for official confirmation)

I think it would be good to use existing feedback and do extra research (wiki, overpass-turbo link above) to make proposition exactly on what elements would the quest trigger and would would it tag / what other answers would it have (i.e. if it has no name, or if the type of monument/memorial is wrong etc).

**Comment by HugoTHOLLON** [comment]
From what I'm seeing I think it might be better to first have a quest that asks the user if it's a monument or a memorial when the monument name is missing. If it is indeed a monument with no name then a second quest could ask for it's name.

**Comment by westnordost** [comment]
(How) can a user distinguish between a memorial and a monument?

**Comment by HugoTHOLLON** [comment]
A memorial is a small structure or statue established to remind people of a person or event. A monument has the same purpose but is a especially large or tall structure (one can go inside, walk on or through it).

In the quest I made there is a hint describing just that. See the PR #6585.

**Comment by danieldegroot2** [comment]
Some historic=city_gate https://taginfo.openstreetmap.org/tags/historic=city_gate also have a name.
(could also be covered by #6612.)

(sidenote: though unable to check right now; see also [Overpass API status](https://wiki.openstreetmap.org/wiki/Overpass_API/status). later / for someone else it might work better)
